app/models/post.rb

```diff
@@ -1,5 +1,13 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   has_many :likes
   has_many :comments
+
+  def increment_posts_counter
+    author.increment!(:posts_counter)
+  end
+
+  def recent_comments
+    comments.order(created_at: :desc).take(5)
```

Comment:
[OPTIONAL] It would be better to use `limit` method rather than `take` method because `take` returns an Array of records while `limit` returns an ActiveRecord Relation that can be chained with other relations. which means:
- `limit(1)` will return an `ActiveRecord::Relation`, meaning you can chain more onto it (e.g. `Account.limit(1).where(...)`).
- `take` will return an Array (either from an already-loaded query or by executing `limit(1)` and returning the array)